contrib/systemd/atomic-openshift-node.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Atomic OpenShift Node
 After=docker.service
-After=atomic-openshift-master.service
```

Comment:
Not needed with the Before: atomic-openshift.node.service on atomic-openshift.master.service